docs/getting_started/bare-metal.rst

```diff
@@ -153,7 +153,7 @@ Create Partion For Docker LVM
 -  please note which partition it is in. So if its partition 5,
    eventually you will need to tell mantl /dev/sda5 for the LVM
 -  you kinda want all your machines to use the same partition because
-   this partition is entered as a system wide variable.
+   this partition is entered as a system wide variable
```

Comment:
If you look at all of the other items in the list, they don't end with punctuation. Removing the period makes it consistent. (it's a copyediting thing.)